<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pm_Dialog_SweepSetup</class>
 <widget class="QWidget" name="Pm_Dialog_SweepSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*******************indoor**************/
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton::checked
{
    color:rgba(87, 87, 87,1);
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QPushButton::!checked
{
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
Pm_LineEdit
{
	background-color:rgba(255,255,255,0.6);
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;QLabel
{
font: 75 13px &quot;Arial&quot;;
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
Pm_LineEdit:focus
{
  outline: none;
  border:2px solid rgba(162,115,0,1);
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}
/***************outdoor*****************/
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget
{
	background-color:rgba(211,211,211,0.9);
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QPushButton::checked
{
    color:rgba(87, 87, 87,1);
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
Pm_LineEdit
{
	color:rgba(162, 115, 0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold; 
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;; 
   color:rgba(46,46,46,1);
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_SweepSetup[status  =  &quot;outdoor&quot;] Pm_LineEdit:disable
{ 
    color:rgba(133,133,133,0.6); 
}
#Pm_Dialog_SweepSetup[status  =  &quot;outdoor&quot;] Pm_LineEdit:focus
 { 
    outline: none;
	border:2px solid rgba(162,115,0,1); 
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}
/***************night*****************/
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
{
	border:1px;
	border-top-style:solid;
	border-top-color:black;
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget
{
	background-color:rgba(208,182,139,0.9);
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton
{
	color:rgba(123, 123, 123,1);
	font: 75 13px &quot;Arial&quot;;
	border-style:flat;
	font-weight:bold;
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton::checked
{
   color:rgba(87, 87, 87,1);
}
/*#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QPushButton::!checked
{
   color:rgba(0, 0, 0, 1)
}
*/
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
Pm_LineEdit
{
	background: rgba(251,220,167,0.60);
   color:rgba(159,98,0,1);
	padding:8px 8px 7px 10px ;
	font: 75 13px &quot;Arial&quot;;
	font-weight:bold;
}
#Pm_Dialog_SweepSetup[status  =  &quot;night&quot;] 
Pm_LineEdit:disable
{
	color:rgba(133,133,133,0.6);
}

#Pm_Dialog_SweepSetup[status  =  &quot;night&quot;] 
Pm_LineEdit:focus
{
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;QLabel
{
	font: 75 13px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#line
{
	background-color:rgba(46,46,46,1);
}
#Pm_Dialog_SweepSetup[status  =  &quot;night&quot;] Pm_LineEdit:disable
{ 
    color:rgba(133,133,133,0.6);
}
#Pm_Dialog_SweepSetup[status  =  &quot;night&quot;] Pm_LineEdit:focus
 { 
	outline: none;
	border:2px solid rgba(162,115,0,1);
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#widget&gt;#pb_Sweep_Once
{
	background-color:rgb(0,0,0);
	color:rgb(255,255,255);
}</string>
  </property>
  <property name="status" stdset="0">
   <string>night</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>252</width>
     <height>648</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="labelSweepType">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>70</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Sweep Type</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_single">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>69</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pb_single:checked
{
	padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_right.png);
	background-position:right;
}

#pb_single:!checked
{
padding: 8px 5px 7px 6px;
	background-image: url(:/pm/image/fre_checked_left.png);
	background-position:right;
}</string>
    </property>
    <property name="text">
     <string>Single</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_conti">
    <property name="geometry">
     <rect>
      <x>47</x>
      <y>100</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*#QPushButton::pressed
{
background-color: rgb(255, 188, 0);
}
*/
#pb_conti:checked
{
	padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_left.png);
}
#pb_conti:!checked
{
padding: 8px 14px 7px 15px;
	background-image: url(:/pm/image/fre_checked_right.png);
}
</string>
    </property>
    <property name="text">
     <string>Conti.</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>46</y>
      <width>212</width>
      <height>2</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_cancel">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/pm/image/btn_close_normal.png);
}
#Pm_Dialog_SweepSetup[status = &quot;indoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/pm/image/btn_close_click.png);
}

#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
background-image: url(:/pm/image/btn_close_normal.png);
}
#Pm_Dialog_SweepSetup[status = &quot;outdoor&quot;]
#pb_cancel:pressed
{
background-image: url(:/pm/image/btn_close_click.png);
}

#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_cancel
{
background-color:rgba(0,0,0,0);
border-image: url(:/pm/image/night/btn_close_normal.png);
}
#Pm_Dialog_SweepSetup[status = &quot;night&quot;]
#pb_cancel:pressed
{
border-image: url(:/pm/image/night/btn_close_click.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>19</y>
      <width>102</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#lblTitle
{
	font: 75 20px &quot;Arial&quot;;
	color:rgba(46,46,46,1);
	line-height:23px;
	font-weight:bold;
}</string>
    </property>
    <property name="text">
     <string>Sweep</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>140</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Time
Interval</string>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="LedTimeInterval">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>1 s</string>
    </property>
    <property name="maxLength">
     <number>13124124</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelMeasureOffset">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>220</y>
      <width>70</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Measure
Offset</string>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="LedMeasureOffset">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>0 dB</string>
    </property>
    <property name="maxLength">
     <number>13124124</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Pm_LineEdit" name="ledDutyCycle">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>100 %</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblDuty">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>180</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Duty cycle</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Pm_LineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">Pm_lineedit.h</header>
   <slots>
    <signal>clicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
